### ISSUE TITLE
problem linking jquery to assets dir on result creation

### ISSUE THREAD

**Issue by gjw** [LateRoomsGroup/moonraker#36]
This is a fresh install with effectively zero content.  Got the same results (with fresh installs of node_modules) while the moonraker dep was set to 0.3.3, 0.3.5, and latest (today).

```
node node_modules/moonraker/bin/moonraker.js

(node:33975) DeprecationWarning: 'GLOBAL' is deprecated, use 'global'
fs.js:1069
  return binding.link(pathModule._makeLong(srcpath),
                 ^

Error: ENOENT: no such file or directory, link 'node_modules/moonraker/node_modules/jquery/jquery.min.js' -> 'results/assets/jquery.min.js'
    at Error (native)
```

No idea how or why it's working for anyone else (assuming it is), but the fix for me is changing builder.js:34 from 
`
var jqueryPath = path.join('node_modules', 'moonraker', 'node_modules', 'jquery', 'jquery.min.js');
`
to
`
var jqueryPath = path.join('node_modules', 'jquery', 'jquery.min.js');
`

(<root>/node_modules/moonraker does not contain a node_modules dir)

**Comment by karlatkinson** [comment]
This is an npm 3 issue as it's expecting the jquery package to be within a moonraker node_modules folder within the node_modules folder of your node application.
npm 3 has a flattened approach to sub-dependencies hence why your fix worked.

We'll get a fix so it works for both sorted.

**Comment by karlatkinson** [comment]
This should be fixed in 0.4.0
